docs/modules/build-system/overview.md

```diff
@@ -0,0 +1,17 @@
+# Build System Feature Module
+
+This feature module contains a default implementation and tooling in the Unity Editor for building your GDK for Unity workers.
+
+> **Note:** The Build System Feature Module does not have an integration for building non-Unity workers at this time.
+
+## Installation
+
+### 1. Add the package
```

Comment:
if there is only one step, should we enumerate it? ;)

Reply:
Mostly just consistency across the feature modules but can just remove it